cds/modules/records/serializers/schemas/common.py

```diff
@@ -166,5 +166,30 @@ class AlternateIdentifiersSchema(StrictKeysSchema):
 
 class CurationSchema(StrictKeysSchema):
     """Curation schema."""
+
     legacy_report_number = fields.Str()
     department = fields.Str()
+    volumes = fields.List(fields.Str())
+
+
+class AdditionalTitlesSchema(Schema):
+    """Additional titles schema."""
+
+    title = fields.Str()
+    type = fields.Str()
+    lang = fields.Str()
+
+
+class AdditionalDescriptionsSchema(Schema):
+    """Additional descriptions schema."""
+
+    description = fields.Str()
+    type = fields.Str()
+    lang = fields.Str()
+
+
+class RelatedIdentifiersSchema(Schema):
+    identifier = fields.Str()
+    scheme = fields.Str()
+    relation_type = fields.Str()
+    resource_type = fields.Str()
```

Comment:
Minor: We should mark as required here as well, the fields that are mandatory, as marked in the schema https://github.com/CERNDocumentServer/cds-videos/pull/2036/files#diff-1dca5dfb1fb3b354ea6b97b94d0e442c9092430a9e52080c53340e8455828407R845